### DISCUSSION TITLE
[Help]Vertical Menu allow only 1 parent open at the time

### DISCUSSION THREAD

**Discussion by lenamtl** [ColorlibHQ/AdminLTE#4614] [General]
Hi,

Sidebar Vertical menu
I'm looking for a way to allow only one parent to be opened.
Any clue?
I use to do this for the first menu version

**Comment by dfsmania** [comment]
I'm not sure what are you looking for, but I think it will be a little hard to implement something like that. Assuming you have a menu defined like the next one:

```bash
> Level 1
  > Level 2
  > Level 2
  > Level 2
    > Level 3
    > Level 3
```

When you expand the latest `Level 2` element, you can't expect to collapse the outer `Level 1` to only see something like next examples:

```bash
> Level 1
  > Level 2
    > Level 3
    > Level 3
```
or just...
```bash

> Level 2
  > Level 3
  > Level 3
```

If you collapse the outer `Level 1` element, all his children will be collapsed too.

Currently, the option you have available is to use the [accordion configuration](https://adminlte.io/docs/3.0/javascript/treeview.html). If I misunderstood your goal, please explain with an example...

**Comment by lenamtl** [comment]
Here is the scenario
Let say 1 have 2 menus with sub-menu

-Contacts
  --Reports 
  --Quotes

-Sales
  --Service 1
  --Service 2

The way it work now
If I click on Contact it open the Contact sub-menu
If I click on Sales it open the Sales sub-menu and Contact sub-menu is still open, **both are open**.

The way I want
If I click on Contact it open the Contact sub-menu.
If I click on Sales it open the Sales sub-menu and **close the Contact sub-menu**.

**Comment by dfsmania** [comment]
@lenamtl , if we are talking about version `v3.2.0`, you need to add `data-accordion="true"` to element `ul` with `data-widget="treeview"`:

```html
<ul data-widget="treeview" data-accordion="true">...</ul>
```
or remove `data-accordion` attribute, since the default value is `true`.
```html
<ul data-widget="treeview">...</ul>
```
Check the [accordion configuration](https://adminlte.io/docs/3.0/javascript/treeview.html)
